src/pyrqlite/cursors.py

```diff
@@ -97,7 +97,7 @@ def _substitute_params(self, operation, parameters):
         param_matches = 0
 
         qmark_re = re.compile(r"(\?)")
-        named_re = re.compile(r"(:{1}[a-zA-Z]+?\b)")
+        named_re = re.compile(r"(:{1}[a-zA-Z_]+?\b)")
```

Comment:
Maybe we could also support $ since I found this:
https://github.com/sqlite/sqlite/blob/master/test/e_expr.test#L546

    Identifiers in SQLite consist of alphanumeric, '_' and '$' characters

Reply:
Also we could possibly support `@` instead of colon:
https://github.com/sqlite/sqlite/blob/master/test/e_expr.test#L559